scripts/user_retirement/utils/helpers.py

```diff
@@ -152,69 +144,13 @@ def _setup_all_apis_or_exit(fail_func, fail_code, config):
         lms_base_url = config['base_urls']['lms']
         ecommerce_base_url = config['base_urls'].get('ecommerce', None)
         credentials_base_url = config['base_urls'].get('credentials', None)
-        segment_base_url = config['base_urls'].get('segment', None)
         license_manager_base_url = config['base_urls'].get('license_manager', None)
         client_id = config['client_id']
         client_secret = config['client_secret']
-        braze_api_key = config.get('braze_api_key', None)
-        braze_instance = config.get('braze_instance', None)
-        amplitude_api_key = config.get('amplitude_api_key', None)
-        amplitude_secret_key = config.get('amplitude_secret_key', None)
-        salesforce_user = config.get('salesforce_user', None)
-        salesforce_password = config.get('salesforce_password', None)
-        salesforce_token = config.get('salesforce_token', None)
-        salesforce_domain = config.get('salesforce_domain', None)
-        salesforce_assignee = config.get('salesforce_assignee', None)
-        segment_auth_token = config.get('segment_auth_token', None)
-        segment_workspace_slug = config.get('segment_workspace_slug', None)
-        hubspot_api_key = config.get('hubspot_api_key', None)
-        hubspot_aws_region = config.get('hubspot_aws_region', None)
-        hubspot_from_address = config.get('hubspot_from_address', None)
-        hubspot_alert_email = config.get('hubspot_alert_email', None)
-
-        for state in config['retirement_pipeline']:
-            for service, service_url in (
-                ('BRAZE', braze_api_key),
-                ('AMPLITUDE', amplitude_api_key),
-                ('ECOMMERCE', ecommerce_base_url),
-                ('CREDENTIALS', credentials_base_url),
-                ('SEGMENT', segment_base_url),
-                ('HUBSPOT', hubspot_api_key),
-            ):
-                if state[2] == service and service_url is None:
-                    fail_func(fail_code, 'Service URL is not configured, but required for state {}'.format(state))
 
+        # Load any Open edX service Apis that are configured
         config['LMS'] = LmsApi(lms_base_url, lms_base_url, client_id, client_secret)
```

Comment:
Open question whether we want to make the 1st party APIs function the same way, but for the sake of simplicity I've left them for now.